sympy/integrals/tests/test_laplace.py

```diff
@@ -94,7 +94,7 @@ def test_laplace_transform():
     assert (LT(exp(-2*t**2), t, s) ==
             (sqrt(2)*sqrt(pi)*exp(s**2/8)*erfc(sqrt(2)*s/4)/4, 0, True))
     assert (LT(b*exp(2*t**2), t, s) ==
-            (b*LaplaceTransform(exp(2*t**2), t, s), -oo, True))
+            (b * (oo * sign(exp(-s**2 / 8)) + (sqrt(2) * sqrt(pi) * exp(-s**2 / 8) * erfi(s * sqrt(2) / 4)) / 4), -oo, True))
```

Comment:
Why is there an infinity here? I think this should remain unevaluated.

Reply:
The closed form of LT exp(2*t**2) is:

![image](https://github.com/user-attachments/assets/c7c84952-9a40-4c2d-b630-1f5c065f075e)

I don't know why did an infinity get added instead of the correct term.

Reply:
There should be a way to fix it but I am not deep inside the laplace transform code.

Reply:
It should be caused by the laplace integral definition getting simplified into one exponential and then solved as an erf term.

Reply:
Okay so I believe that Mathematica's output is faulty bc this LT does not actually converge in any possible way. I couldn't find a way to show that the integral diverges, so I put the condition on the laplace transform function to check whether the final answer has an infinity or not, and if yes, it assumes that the form does not converge.